playlist/player.go

```diff
@@ -141,8 +150,11 @@ func (sp SchedulePlayer) playTodaysCombos(combos []Combo) {
 		log.Println("Playing combo...")
 		sp.playCombo(combos[count])
 		count = (count + 1) % numberCombos
-		nextComboStart = sp.time.Now().Add(sp.createWindow(combos[count]).Until())
-		log.Printf("Next combo start %v", nextComboStart)
+		if count > 1 {
+			nextComboStart = sp.time.Now().Add(sp.createWindow(combos[count]).Until())
+		} else {
+			nextComboStart = sp.time.Now().Add(sp.createWindow(combos[count]).Until())
+		}
```

Comment:
Why are both branches of this if else the same??

Reply:
Good spotting....